metadata-service/configuration/src/main/resources/application.yaml

```diff
@@ -115,6 +115,7 @@ graphService:
 searchService:
   resultBatchSize: ${SEARCH_SERVICE_BATCH_SIZE:100}
   enableCache: ${SEARCH_SERVICE_ENABLE_CACHE:false}
+  enableEviction: ${SEARCH_SERVICE_ENABLE_CACHE_EVICTION:false}
```

Comment:
Once included in a release, we'll want to update our helm chart to support enabling it.